packages/purpose-process/src/services/purposeService.ts

```diff
@@ -837,6 +840,41 @@ export function purposeServiceBuilder(
         isRiskAnalysisValid,
       };
     },
+    async retrieveRiskAnalysisConfigurationByVersion({
+      eserviceId,
+      riskAnalysisVersion,
+      organizationId,
+    }: {
+      eserviceId: EServiceId;
+      riskAnalysisVersion: string;
+      organizationId: TenantId;
+    }): Promise<RiskAnalysisFormRules> {
+      logger.info(
+        `Retrieve version ${riskAnalysisVersion} of risk analysis configuration`
+      );
+
+      const eservice = await retrieveEService(eserviceId, readModelService);
+      const tenant = await getInvolvedTenantByEServiceMode(
+        eservice,
+        organizationId,
+        readModelService
+      );
+
+      assertTenantKindExists(tenant);
+
+      const riskAnalysisFormConfig = riskAnalysisFormRules[tenant.kind].find(
```

Comment:
In Scala there's also the error `RiskAnalysisConfigForTenantKindNotFound` that's thrown when a config is not found for that `TenantKind` but I don't think that's useful since if the kind is not one of those we would get an error in its parsing.